Dockerfile

```diff
@@ -274,20 +275,19 @@ USER root
 RUN curl -o- -L https://yarnpkg.com/install.sh > /usr/local/bin/yarn-installer.sh
 
 # Install node.js
+ENV NVM_VERSION=0.35.3
 USER buildbot
 RUN git clone https://github.com/creationix/nvm.git ~/.nvm && \
     cd ~/.nvm && \
-    git checkout v0.34.0 && \
+    git checkout v"$NVM_VERSION" && \
     cd /
 
 ENV ELM_VERSION=0.19.0-bugfix6
-ENV YARN_VERSION=1.13.0
-
-ENV NETLIFY_BUILD_NODE_VERSION="12.16.1"
+ENV YARN_VERSION=1.22.4
+ENV NODE_VERSION=12.16.2
```

Comment:
Why do you not use the latest version 12.16.3?

Reply:
Or even better find a way to use the latest version node version in the dockerfile.
My problem currently is, that we defined the node version with 12 in the .npmrc file.
That means in the docker we install a node version and in the next step my Netlify build will install the latest node again.

Reply:
Or another idea why do you not check if a NODE_VERSION env variable is set and use this value?